markdown.dtx

```diff
@@ -10148,6 +10195,37 @@ following text except for the filename, which may differ:
 %</tex>
 %<*manual-tokens>
 % \fi
+% \begin{markdown}
+
+#### YAML Metadata Renderers
+The \mdef{markdownRendererSetJekyllData} macro represents a fenced code
+block which will not be read verbatim but allows the user to set title information 
+or metadata. It can be extended to create arbitrary variables using the 
+YAML-Syntax. This macro will only be produced, when the \Opt{jekyllData} option is
+enabled. The macro receives two argument where the first one represents the variable
+name whereas the second contains the data.
+
+% \end{markdown}
+%
+% \iffalse
+%</manual-tokens>
+%<*tex>
+% \fi
+%
+%  \begin{macrocode}
+\def\markdownRendererSetJekyllData{%
+  \markdownRendererSetJekyllDataPrototype}%
+\def\markdownRendererSetJekyllDataBegin{%
+  \markdownRendererSetJekyllDataBeginPrototype}%
+\def\markdownRendererSetJekyllDataEnd{%
+  \markdownRendererSetJekyllDataEnd}%
+%    \end{macrocode}
+% \par
+%
+% \iffalse
+%</tex>
+%<*manual-tokens>
```

Comment:
This is ok for now, but we will need to add some examples after we're happy with the implementation.

Reply:
Fixed in 0c96c8a.